packages/ndla-ui/src/Embed/AudioEmbed.tsx

```diff
@@ -42,9 +29,7 @@ const renderMarkdown = (text: string) => {
   return <span dangerouslySetInnerHTML={{ __html: rendered }} />;
 };
 
-const AudioEmbed = ({ embed, articlePath }: Props) => {
```

Comment:
articlePath brukes fremdeles for podkaster: https://test.ndla.no/subject:6e2e2319-cb8a-4dd2-b382-e30f001633bb/topic:94de9967-1492-479b-b0a8-6764a616b50d/topic:954b3d73-7652-447c-9fe5-e326ded1ae9a/resource:7ad9f2f6-a35b-4bce-b7fe-a19f60337bda